frame/support/test/tests/pallet.rs

```diff
@@ -881,7 +881,7 @@ fn pallet_expand_deposit_event() {
 
 #[test]
 fn pallet_new_call_variant() {
-	Call::Example(pallet::Call::new_call_variant_foo(3, 4));
+	pallet::Call::<Runtime>::new_call_variant_foo(3, 4);
```

Comment:
Based on `frame/support/test/tests/issue2219.rs` why no `let x = ...; drop(x)` here?

Reply:
clippy was complaining before that the Call::Example( bit was not relevant to anything as nothing was then using it. I don't think we need to add any drops here.